From c5f91f08eb787ea5851019468e8ce800833f7853 Mon Sep 17 00:00:00 2001
From: root <root@buildhost>
Date: Sun, 16 Jul 2023 12:14:31 +0200
Subject: [PATCH] arm: dts: add sun8i-h3-fingbox-v1.dts

---
 arch/arm/boot/dts/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 5b4d1386d4778..dfca1b683c56e 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -1370,6 +1370,7 @@ dtb-$(CONFIG_MACH_SUN8I) += \
 	sun8i-h3-bananapi-m2-plus.dtb \
 	sun8i-h3-bananapi-m2-plus-v1.2.dtb \
 	sun8i-h3-beelink-x2.dtb \
+	sun8i-h3-fingbox-v1.dtb \
 	sun8i-h3-libretech-all-h3-cc.dtb \
 	sun8i-h3-mapleboard-mp130.dtb \
 	sun8i-h3-nanopi-duo2.dtb \
-- 
2.25.1

