From be605a9e5229a881759f5a0a7237bddc2a0178ba Mon Sep 17 00:00:00 2001
From: root <root@buildhost>
Date: Wed, 5 Jul 2023 14:01:18 +0200
Subject: [PATCH] arm: dts: add sun5i-a13-q8a-tablet.dts

---
 arch/arm/boot/dts/allwinner/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/allwinner/Makefile b/arch/arm/boot/dts/allwinner/Makefile
index efe4152e5846d..5b4d1386d4778 100644
--- a/arch/arm/boot/dts/allwinner/Makefile
+++ b/arch/arm/boot/dts/allwinner/Makefile
@@ -1293,6 +1293,7 @@ dtb-$(CONFIG_MACH_SUN5I) += \
 	sun5i-a13-olinuxino-micro.dtb \
 	sun5i-a13-pocketbook-touch-lux-3.dtb \
 	sun5i-a13-q8-tablet.dtb \
+	sun5i-a13-q8a-tablet.dtb \
 	sun5i-a13-utoo-p66.dtb \
 	sun5i-gr8-chip-pro.dtb \
 	sun5i-gr8-evb.dtb \
-- 
2.25.1

